tests/filtertransformers/test_mongo.py

```diff
@@ -226,6 +226,63 @@ def test_operators(self):
         # OPTIONAL
         # self.assertEqual(self.transform("((NOT (_exmpl_a>_exmpl_b)) AND _exmpl_x>0)"), {})
```

Comment:
This still doesn't work?

Reply:
We don't have this property comparison thing yet right? I think we need to wait for clarification in the spec before deciding how to interpret these. (See note here https://github.com/Materials-Consortia/OPTiMaDe/blob/develop/optimade.rst#numeric-and-string-comparisons)

Reply:
Right!